TestingStaff/tstStock.cs

```diff
@@ -47,11 +65,107 @@ public void AttributeQuantityAvailableOK()
             Assert.IsTrue(product.QuantityAvailable >= 0 && product.QuantityAvailable <=50);
         }
 
+        //Test Restock Date
         [TestMethod]
         public void AttributeRestockDateOK()
         {
             clsStock product = new clsStock();
             Assert.IsNotNull(product.RestockDate);
         }
+
+        //Create Find Porduct ID Test
+        [TestMethod]
+        public void TestProductIdFound()
+        {
+            clsStock product = new clsStock();
+            Boolean Found = false;
+            Boolean OK = true;
+            Int32 ProductId = 2;
+            Found = product.Find(ProductId);
+            if(product.ProductId != 2)
+            {
+                OK = false;
+            }
+            Assert.IsTrue(OK);
+        }
+
+        [TestMethod]
+        public void TestProductNameFound()
+        {
+            clsStock product = new clsStock();
+            Boolean Found = false;
+            Boolean OK = true;
+            Int32 ProductId = 2;
+            Found = product.Find(ProductId);
+            if(product.ProductName != "Product Name")
```

Comment:
These test shouldn't use custom booleans, instead the test should be `Assert.IsTrue(product.ProductName != "Product Name")`